OrangeHRMTestFramework/PageObjects/OrangeHRM/DataGrids/GenericDataGrids.cs

```diff
@@ -0,0 +1,11 @@
+﻿using OrangeHRMTestFramework.PageObjects.OrangeHRM.Filters;
+
+namespace OrangeHRMTestFramework.PageObjects.OrangeHRM.DataGrids
+{
+    public static class GenericDataGrids
```

Comment:
Yep, this pattern works from code perspective but please use the pattern for Pages/Popups entities only.
Grids, filters etc. are more like "elements" on the pages/popup. So you can create fields or properties in Page/Popup classes for filter/grid etc and call method by chain -> {page instance}.{property of grid}.{grid method}

Please make the same changes for filters

Reply:
Sounds great! Fixed

Reply:
Please do the same for Forms

Reply:
Fixed